### PR TITLE
restyle LR drawer

### LINKED ISSUE #2253: Change display of course/program link on Drawer
Change the layout of the `ExpandedLearningResourceDisplay` external link from:

Screen Shot 2019-09-27 at 11 25 52 AM

to:

Screen Shot 2019-09-27 at 11 27 02 AM

Design at https://projects.invisionapp.com/d/main#/console/17477251/376322443/preview

### LINKED ISSUE #2466: Learn: update drawer styling
#### Designs and Mockups

[InVision](https://projects.invisionapp.com/d/main#/console/17477251/392253808/preview)

#### Acceptance Criteria:

- [x] #2253 change course button and logo
- [x] Update font and size of title
- [x] Update text styling of description
- [x] Hide "Read More" link when there is nothing more to read
- [x] remove "Info" header"
- [x] Change styling of metadata section
  - [x] text color
  - [x] two-column layout
  - [x] change "topics" label to "subjects" 
  - [x] remove button styling for topics/subjects

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?

closes #2466
closes #2253

#### What's this PR do?

This updates the display of learning resources in the drawer to match our latest designs. I also updated the display of the logo for the LR platform.

I have a separate PR open (#2484) which fixes the `TruncatedText` component.

Basically, this mainly is just restyling things, but there are a few more major parts:

- I pulled out hte styling for the expanded learning resource display from `course.scss` to a new file, `learning-resource-drawer.scss`, which has only the styling for the drawer in it.
- I also finally updated a lot of selectors like were of the form `course-foo` to be just `foo`, since those selectors were originally written when we were only showing courses and we're now showing bootcamps and programs and so on.

#### How should this be manually tested?

look at all kinds of learning resources and make sure that nothing is broken. make sure that the design is good based on the discussion on the issue, that things are readable on all screen widths and so on.

#### Screenshots (if appropriate)

ffdasdfbootcampasdfasfdvideoasdfasdf
ffdasdfbootcampasdfasfdvideo
ffdasdfbootcampasdfasfd
ffdasdfbootcamp
 ⛔ 🚧 🛑 📣

### CODE REVIEW

File: static/js/components/ExpandedLearningResourceDisplay.js [hunk 1]
```diff
@@ -126,24 +131,31 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
...
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {`Take ${capitalize(object.object_type)}`}
+              </a>
+              <div className="platform-logo">
+                <span>on</span>
+                <img
+                  src={
+                    platformLogos[
```
[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: @Ferdi what should we do in this case?

File: static/js/components/ExpandedLearningResourceDisplay.js [hunk 2]
```diff
@@ -152,82 +164,76 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
...
+                    "Privacy:",
+                    capitalize(object.privacy_level)
+                  )}
+                  {lrInfoRow("view_list", "Items in list:", object.items.length)}
+                </React.Fragment>
+              ) : null}
+          </div>
         </div>
       </div>
       {hasCourseList(object.object_type) ? (
```
[the remark this answers is not in the document]
Comment (mbertrand) [reply static/js/components/ExpandedLearningResourceDisplay.js]: PS Also just noticed that the star has been replaced with a bookmark icon in the design, maybe that's for a separate PR?

[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: where is that invision design for "Similar Learning Resources"? I see this one which matches what we have implemented for "Learning Resources in this List"

https://projects.invisionapp.com/d/main?origin=v7#/console/18704964/392253636/preview?scrollOffset=877

File: static/js/lib/constants.js [hunk 3]
```diff
@@ -27,6 +27,14 @@ export const platforms = {
   youtube:      youtubePlatform
 }
 
+export const platformLogos = {
+  [ocwPlatform]:          "/static/images//mit-ocw-logo-square.png",
+  [edxPlatform]:          "/static/images/edx_logo.png",
+  [bootcampsPlatform]:    "/static/images/mit-bootcamp-logo.png",
+  [micromastersPlatform]: "/static/images/mit-micromasters-logo.png",
+  [xproPlatform]:         "/static/images/mit-xpro-logo.png"
+}
```
[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/lib/constants.js]: ah, I don't have those! @abdulkdawson do you have a logo asset for open learning library?

File: static/js/components/ExpandedLearningResourceDisplay.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mbertrand) [reply static/js/components/ExpandedLearningResourceDisplay.js]: Here is the link: https://projects.invisionapp.com/d/main#/console/17477251/392253808/preview

It's from the top of the description for issue #2466

Not sure which is most up-to-date

[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: ah, @abdulkdawson which of these two header styles should we go with?

File: static/js/components/ExpandedLearningResourceDisplay.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: I could default to `object.platform || object.offered_by`

File: static/js/components/ExpandedLearningResourceDisplay.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: bump :)

**Comment by aliceriot** [comment]
@abdulkdawson @Ferdi @pdpinch still waiting for a response for some of the questions on this PR

**Comment by abdulkdawson** [comment]
@aliceriot The one at the bottom is the correct header.

**Comment by abdulkdawson** [comment]
@aliceriot They should look like this.
Screen Shot 2019-12-20 at 11 43 19 AM
Screen Shot 2019-12-20 at 11 43 13 AM

File: static/js/components/ExpandedLearningResourceDisplay.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aliceriot) [reply static/js/components/ExpandedLearningResourceDisplay.js]: @mbertrand will object always have an 'offered_by' if they lack a platform? if so I'll just go with the above I think

[the remark this answers is not in the document]
Comment (mbertrand) [reply static/js/components/ExpandedLearningResourceDisplay.js]: I think programs/bootcamps/courses should have at least 1 or the other, though it's also possible for a resource to have 2+ offered_by values too.

[the remark this answers is not in the document]
Comment (Ferdi) [reply static/js/components/ExpandedLearningResourceDisplay.js]: sorry, i missed this. we don't programs have "platform" ? Is it hard to add ?

**Comment by abdulkdawson** [comment]
Open_Learning_Logo

This should work.

**Comment by aliceriot** [comment]
@mbertrand ok I think those issues should be resolved now

**Pull request merged**